web/content/release-3.0.adoc

```diff
@@ -0,0 +1,535 @@
+:FOREMAN_VER: 3.0
+:KATELLO_VER: 4.2
+
+= Foreman {FOREMAN_VER} and Katello {KATELLO_VER}
+
+Use the top menu bar for navigation in the following guides:
+
+* link:/{FOREMAN_VER}/Planning_Guide/index-foreman-el.html[Planning for Foreman]
+* link:/{FOREMAN_VER}/Release_notes/index-foreman-el.html[Release notes for Foreman]
+* link:/{FOREMAN_VER}/Release_notes/index-katello.html[Release notes for Katello]
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-foreman-el.html[Foreman Quickstart Guide on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-foreman-deb.html[Foreman Quickstart Guide on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Quickstart_Guide/index-katello.html[Katello Quickstart Guide on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server_on_Red_Hat/index-foreman-el.html[Installing Foreman on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server_on_Red_Hat/index-katello.html[Installing Katello on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Proxy_on_Red_Hat/index-foreman-el.html[Installing Smart Proxy on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server_on_Debian/index-foreman-deb.html[Installing Foreman on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Installing_Proxy_on_Debian/index-foreman-deb.html[Installing Smart Proxy on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Upgrading_and_Updating/index-foreman-el.html[Upgrading and Updating Foreman]
+* link:/{FOREMAN_VER}/Upgrading_and_Updating/index-katello.html[Upgrading and Updating Katello]
+* link:/{FOREMAN_VER}/Deploying_on_AWS/index-foreman-el.html[Deploying Foreman on AWS (reference guide)]
+* link:/{FOREMAN_VER}/Configuring_Load_Balancer/index-foreman-el.html[Configuring Smart Proxies with a Load Balancer]
+* link:/{FOREMAN_VER}/Provisioning_Guide/index-foreman-el.html[Provisioning Guide]
+* link:/{FOREMAN_VER}/Content_Management_Guide/index-katello.html[Content Management Guide]
+* link:/{FOREMAN_VER}/Configuring_Ansible/index-foreman-el.html[Configuring Foreman to use Ansible]
+* link:/{FOREMAN_VER}/Managing_Hosts/index-foreman-el.html[Managing Hosts Guide]
+* link:/{FOREMAN_VER}/Administering_Red_Hat_Satellite/index-foreman-el.html[Administering Foreman]
+* link:/{FOREMAN_VER}/Application_Centric_Deployment/index-foreman-el.html[Application Centric Deployment]
+
+== Foreman {FOREMAN_VER} release notes
```

Comment:
Please don't do this. Link to the actual guide we have for it.

Reply:
IMHO this comment is blocking to merging this. I strongly insist we don't duplicate content.

Reply:
Even 3.0? Okay.